### ISSUE TITLE
Support filters inside prop expressions

### ISSUE THREAD

**Issue by wheeljs** [yyx990803/vue#1237]
Thank's for reading.

I'm developing a todoDemo, and I'm trying to use vue 1.0.0-alpha.2 version. When I update sub-compnents props use `prop-` prefix, I got a invalid expression error. 
I want to use a filter with a computed property that filter by todo types.
vue config:
```javascript
let filters = {
  all: (todos) => todos.filter((todo) => todo.isDelete === false),
  active: (todos) => todos.filter((todo) => todo.isFinished === false && todo.isDelete === false),
  finished: (todos) => todos.filter((todo) => todo.isFinished === true && todo.isDelete === false)
};

let vm = new Vue({
  data: () => ({
    todos: [
      {
        content: 'hello world'
      },
      // some todos from localStorage
    ],
    filter: 'all',
    query: '' // todo content filter.
  }),
  computed: {
    filteredTodos() {
      return filters[this.filter](this.todos);
    }
  },
  components: {
    'todo-list': compTodoList
  },
  filters: {
    orderTodos(todos) {
      return todos.sort((todo1, todo2) => {
        if (todo1.isDelete === todo2.isDelete) {
          if (todo1.isFinished === todo2.isFinished) {
            return new Date(todo2.lastModified).getTime()
                    - new Date(todo1.lastModified).getTime();
          }
          return todo1.isFinished - todo2.isFinished;
        }
        return todo1.isDelete - todo2.isDelete;
      });
    }
  }
});
```
component:
```html
<todo-list v-if="todos.length", prop-todos="filteredTodos | filterBy query in 'content' | orderTodos"></todo-list>
```

After using webpack to package a `.vue` file into `build.js` file. Open it in browser I had get a error from developer tools.
```
[Vue warn]: Invalid expression. Generated function body:  scope.filteredTodos|scope.filterByqueryin'content'|scope.orderTodos
```
But when I call vue instance's `$eval` method and pass string `filteredTodos | filterBy query in "content" | orderTodos` as parameter, I will get correct result.

thx.

**Comment by yyx990803** [comment]
Filters are currently not supported inside prop expressions, but they probably should be.

**Comment by wheeljs** [comment]
OK, thx for reading and receiving me.

**Comment by yyx990803** [comment]
Implemented in a1357e9
